hazelcast/client.py

```diff
@@ -373,6 +378,21 @@ def get_topic(self, name: str) -> Topic[MessageType]:
         """
         return self._proxy_manager.get_or_create(TOPIC_SERVICE, name)
 
+    def create_vector_collection(self, name: str, indexes: typing.List[IndexConfig]) -> None:
```

Comment:
this is adding vector collection config, not creating vector collection, so maybe rename the method

Reply:
That makes sense.
Renamed it to `create_vector_collection_config` at:
https://github.com/hazelcast/hazelcast-python-client/pull/674/commits/3c27230ad7d9777be1698db0d6ceb00121844ba8